### DISCUSSION TITLE
WIRE:STREAM not work

### DISCUSSION THREAD

**Discussion by wilianmaique** [livewire/livewire#9088] [7. Bugs (archived)]
### Livewire version

v3.5.17

### Laravel version

v11.31

### Which PHP version are you using?

PHP 8.3

### What is the problem?

'wire:stream' doesn't work, it just freezes.
I'm using 'laragon' 'localhost'

Follow video
[https://imgur.com/cFKvF4y](https://imgur.com/cFKvF4y)

### Code snippets

```
<?php

namespace App\Livewire\Panel\Admin\Pages;

use Livewire\Component;

class Testes extends Component
{
  public $start = 3;

  public function run()
  {
    for ($i = 0; $i < 3; $i++) {
      $this->stream('count', $this->start, true);

      usleep(1000000);

      $this->start = $this->start - 1;
    }
  }

  public function render()
  {
    return view('livewire.panel.admin.pages.testes');
  }
}
```

```
<div>
  <x-ts-button wire:click="run">Stream</x-ts-button>

  <div class="py-5">
    Count: <span wire:stream="count">{{ $start }}</span>
  </div>
</div>
```

### Wirebox URL

https://wirebox.app/

### How do you expect it to work?

'wire:stream' doesn't work, it just freezes.
I'm using 'laragon' 'localhost'

Follow video
[https://imgur.com/cFKvF4y](https://imgur.com/cFKvF4y)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by koltejas** [comment]
@calebporzio can you please look into this. Same issue for me too. Failed on production server too. 
Apache, Php 8.3.8, livewire 3.5.
It appears to be happening from v3 [as discussed here](https://github.com/livewire/livewire/discussions/7047)

**Reply by wilianmaique** [reply]
> @calebporzio can you please look into this. Same issue for me too. Failed on production server too. Apache, Php 8.3.8, livewire 3.5. It appears to be happening from v3 [as discussed here](https://github.com/livewire/livewire/discussions/7047)

I also opened this discussion, but nothing yet: [https://laracasts.com/discuss/channels/livewire/wirestream-not-work](https://laracasts.com/discuss/channels/livewire/wirestream-not-work)
